examples/calc/calc.ts

```diff
@@ -1,6 +1,6 @@
-import { ops_list, ops_symbols_list, argvType } from './ops.ts';
+import {ops_list, ops_symbols_list} from './ops';
 
-export function calc(argv: argvType): string {
+export function calc(argv: {[x: string]: string[]}): string {
```

Comment:
I guess the comment I had from this morning was just that it's nice to break complex types like this out into a `type Foo =` declaration, or an `interface`. Then you can also name it to give others an idea of what you had in mind with it.

Reply:
Good call, I actually had an `interface` for this defined in `.ops.ts` but was getting an error because after transpiling the interface didn't exist.  This is a bug I will address but for now I'll move that interface to `calc.ts` itself.